log10/feedback/feedback.py

```diff
@@ -111,3 +160,40 @@ def _get_feedback_list(offset, limit, task_id):
         return []
 
     return feedback_data
+
+
+def _format_graphql_node(node):
+    return {
+        "id": node["id"],
+        "json_values": node["jsonValues"],
+        "task_id": node["task"]["id"],
+        "task_name": node["task"]["name"],
+        "matched_completion_ids": [c["id"] for c in node["completions"]],
+    }
+
+
+def _get_feedback_list_graphql(page, limit, task_id, filter):
+    feedback_data = []
+    current_page = page if page else 1
+    if limit:
+        limit = int(limit)
+
+    try:
+        while True:
+            fetch_limit = limit if limit else 50
+            res = Feedback().list_v2(page=current_page, limit=fetch_limit, task_id=task_id, filter=filter)
+            new_data = res["data"]["organization"]["feedbackV2"]["nodes"]
+            feedback_data.extend(new_data)
+
+            current_fetched = len(new_data)
+            current_page += 1
+
+            if current_fetched < fetch_limit:
```

Comment:
just wonder if there's edge case, like when current_fetched = 50 and fetch_limit = 50, so it would go to next iteration?